lib/vrl/stdlib/src/parse_key_value.rs

```diff
@@ -42,6 +42,11 @@ impl Function for ParseKeyValue {
                 kind: kind::BYTES,
                 required: false,
             },
+            Parameter {
+                keyword: "accept_standalone_key",
+                kind: kind::BOOLEAN,
+                required: false,
+            },
         ]
     }
 
```

Comment:
It's worth adding an example in the `examples` function below this line (can't highlight it because it falls outside the diff range). Those examples are picked up by our integration tests as well.